common/scriptcore/luascript.cpp

```diff
@@ -372,7 +263,14 @@ static void luascript_exec_resource(lua_State *L, const QString &filename)
 
   // We trust that it loads.
   sol::state_view lua(L);
-  lua.script(data.data(), in.fileName().toStdString());
+  auto result = lua.safe_script(data.data(), in.fileName().toStdString());
+  if (!result.valid()) {
+    sol::error err = result;
+    std::cout << "An error (an expected one) occurred: " << err.what()
```

Comment:
Use `qCritical` and friends for error messages like this. A bundled script not loading is quite serious so it should be `qFatal` if you want to check